1906-06-11.xml

```diff
@@ -1375,8 +1376,1163 @@
+                            stadium) representatives of the Governor of Alexandria, the sister
+                            colleges, the press, and various communities. The band of St.
+                            Catherine’s College supplied the musical part of the entertainment, and
+                            successfully performed a choice repertoire, which greatly encouraged the
+                            youthful gymnasts, who looked extremely natty in their tricolour
+                            uniforms. The events were twenty-two, including an amusing sack-race ;
+                            and were divided into field sports and gymnastic exercises. The whole
+                            was under the direction of Mr. H. C. Tookey, who looked every inch an
+                            athlete. He certainly has infused in to the college the essence of
+                            discipline, which is the aim of all athletics, and the rounds of
+                            applause which were showered upon his pupil were no less meant for him.
+                            But the person who, above all, deserves the fullest need of praise is
+                            the indefatigable fere director of the college, to whose efforts in a
+                            large measure is due the success of yesterday’s exhibition. After the
+                            distribution of prizes, the jury presented a silver cup to Mr. H. C.
+                            Tookey, in testimony of their appreciation of his services.</p>
+                    </div>
+                    <div type= "item" scope="advertisement"> 
+                        <head>Commercial and Financial Supplement OF THE  "Egyptian Gazette."                                                     
+                        </head>
+                        <p>The Commercial and Financial Supplement of the "Egyptian Gazette” is
+                            published at midday every Saturday in time for the Austrian Lloyd’s
+                            mail. The supplement contains exhaustive and important reviews of the
+                            cotton, cotton seed, and stock and share markets, with all the latest
+                            statistics up to the evening of the preceding day, complete tabular
+                            forms of the various market fluctuations, and the copies of the official
+                            telegrams of the Liverpool Cotton Association, etc., etc. Subscription
+                            for one year P.T. 100 (inclusive of postage in Egypt. For abroad the
+                            postage is P.T. 10 extra). For further particulars apply to the Manager
+                            “Egyptian Gazette.”</p>
+                    </div>                
+                </div>
+                <cb n="2"/>
+                <div type="section" scope="local">
```

Comment:
This section, which is all reports from the wire services (Reuter and Havas), should be `<div type="section" element="wire">`. Every item within the section can have its own dateline.